support-frontend-static/PROD-deploy.sh

```diff
@@ -0,0 +1,8 @@
+# Run this file to deploy the bucket
```

Comment:
Maybe describe what this bucket is used for as there are a number of other buckets used by the app.

Reply:
good idea, I've done that